cmd/daemon/kubernetes/pods.go

```diff
@@ -0,0 +1,207 @@
+	watcher, err := c.clientset.CoreV1().Pods("").Watch(ctx, metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+
+	for {
+		select {
+		case <-ctx.Done():
+			watcher.Stop()
+		case e, ok := <-watcher.ResultChan():
+			if !ok {
+				return ErrWatcherClosed
+			}
+			if e.Object == nil {
+				continue
+			}
+			pod, ok := e.Object.(*corev1.Pod)
+			if !ok {
+				continue
+			}
+			// Is this a pod managed by the release manager - and does it contain the information needed?
+			if !isPodCorrectlyAnnotated(pod) {
+				continue
+			}
+			// Avoid reporting on pods that has been marked for termination
+			if isPodMarkedForTermination(pod) {
+				continue
+			}
+
+			if isPodInCrashLoopBackOff(pod) {
+				log.Infof("Pod: %s is in CrashLoopBackOff", pod.Name)
+				restartCount := pod.Status.ContainerStatuses[0].RestartCount
+				if math.Mod(float64(restartCount), c.moduloCrashReportNotif) != 1 {
+					continue
+				}
+				var errorContainers []http.ContainerError
+				for _, cst := range pod.Status.ContainerStatuses {
+					if cst.State.Waiting != nil && cst.State.Waiting.Reason == "CrashLoopBackOff" {
+
+						logs, err := getLogs(ctx, c.clientset, pod.Name, cst.Name, pod.Namespace)
+						if err != nil {
+							log.Errorf("Error retrieving logs from pod: %s, container: %s, namespace: %s", pod.Name, cst.Name, pod.Namespace)
+						}
+
+						errorContainers = append(errorContainers, http.ContainerError{
+							Name:         cst.Name,
+							ErrorMessage: logs,
+							Type:         "CrashLoopBackOff",
+						})
+					}
+				}
+
+				err = c.exporter.SendPodErrorEvent(ctx, http.PodErrorEvent{
+					PodName:     pod.Name,
+					Namespace:   pod.Namespace,
+					Errors:      errorContainers,
+					ArtifactID:  pod.Annotations["lunarway.com/artifact-id"],
+					AuthorEmail: pod.Annotations["lunarway.com/author"],
+				})
+				if err != nil {
+					log.Errorf("error SendCrashLoopBackOffEvent")
+				}
+				continue
+			}
+
+			if isPodInCreateContainerConfigError(pod) {
+				log.Infof("Pod: %s is in CreateContainerConfigError", pod.Name)
+				// Determine which container of the deployment has CreateContainerConfigError
+				var errorContainers []http.ContainerError
+				for _, cst := range pod.Status.ContainerStatuses {
+					if cst.State.Waiting != nil && cst.State.Waiting.Reason == "CreateContainerConfigError" {
+						errorContainers = append(errorContainers, http.ContainerError{
+							Name:         cst.Name,
+							ErrorMessage: cst.State.Waiting.Message,
+							Type:         "CreateContainerConfigError",
+						})
+					}
+				}
+
+				err = c.exporter.SendPodErrorEvent(ctx, http.PodErrorEvent{
+					PodName:     pod.Name,
+					Namespace:   pod.Namespace,
+					Errors:      errorContainers,
+					ArtifactID:  pod.Annotations["lunarway.com/artifact-id"],
+					AuthorEmail: pod.Annotations["lunarway.com/author"],
+				})
+				if err != nil {
+					log.Errorf("error SendCreateContainerConfigError")
```

Comment:
The error is missing from this log

```suggestion
					log.Errorf("Failed to send create container config error: %v", err)
```